### DISCUSSION TITLE
Insert cursor appearing?

### DISCUSSION THREAD

**Discussion by jji05** [zed-industries/zed#9945] [Feature Requests]
This could be a duplicate question and was unsure what this feature was. (i've attempted to search for the keyword "cursor" although nothing similar appeared)

This often appears with all the files that I open and i'm unable to type where the cursor is and every key stroke I press causes the cursor to jump to another position. I find this very frustrating, is there some way to disable this or to return the cursor to the regular  mode? 
<img width="134" alt="image" src="https://github.com/zed-industries/zed/assets/61447509/a72ca8db-a445-41b9-8d17-c91599c9be66">

I'm expecting something like this so I can type...
<img width="112" alt="Screenshot 2024-03-29 at 7 33 04 PM" src="https://github.com/zed-industries/zed/assets/61447509/11804a87-f62c-4bdd-9373-612735beb7ee">

**Comment by bennetbo** [comment]
Maybe im not getting your point, but it sounds like you are using vim mode but actually don't want to?
Can you check your settings (Cmd+Shift+P -> "zed: open settings") and see if there is `vim_mode` enabled? If so make sure to set it to false

**Reply by jji05** [reply]
Thank you so much! That fixed my issue 😂 it was vim that was causing the small trouble I listed.
